packages/web-components/src/components/rux-classification-marking/rux-classification-marking.scss

```diff
@@ -23,7 +23,7 @@
     display: flex;
     justify-content: center;
     align-items: center;
-    min-height: 26px;
+    min-height: var(--spacing-6);
```

Comment:
Verifying @FMorrison87, this is the intended change to review?

Reply:
Correct!